### DISCUSSION TITLE
Login into multiple sites

### DISCUSSION THREAD

**Discussion by wm-simon** [statamic/cms#8147] [Q&A]
Have a multi-site and use the `User:Login_Form` tag. The users should login to multiple sites with this form. I've tried `site="*"` and `site="site1|site2`. I couldn't find something in the docs to this. Right now, the login works just for the current site. Perhaps it's not supported or there is another way. Can someone help with this?

**Comment by jasonvarga** [comment]
Logging in would log you into all sites. The multisite feature doesn't separate them. Or at least, it isn't supposed to.

Are you using different domains?

**Comment by wm-simon** [comment]
Yes, I use different Domains. I have a site with secured content, where users can login. When they are logged in, the other sites should show secured content from the site, where they can login.

**Comment by jasonvarga** [comment]
Ah, okay. You could try setting the session domain in your `.env` file.

```
SESSION_DOMAIN=defaultdomain.com
```

Then the sessions will be shared as if they were all on the same domain.

**Comment by wm-simon** [comment]
I can not login anymore, not in the cp and not at the frontend. I've cleared the cache, but without success.
This are my settings in the `.env`
```
APP_URL_WERBEAGENTUR="http://2023.domain1.test"
APP_URL_FOTOGRAFIE="http://2023.domain2.test"
APP_URL_ACCOUNT="http://2023.domain3.test"
SESSION_DOMAIN=2023.domain1.test
```

**Comment by wm-simon** [comment]
I've changed the SESSION_DOMAIN to the `APP_URL_ACCOUNT`. Now i can login with the APP_URL_ACCOUNT to the cp and the frontend. But the frontend on the first site (APP_URL_WERBEAGENTUR) does still not recognize the login.
